docs/hub/storage-limits.md

```diff
@@ -78,3 +101,13 @@ For hosting large datasets on the Hub, we require the following for your dataset
 - Avoid the use of custom loading scripts when using datasets. In our experience, datasets that require custom code to use often end up with limited reuse.
 
 Please get in touch with us if any of these requirements are difficult for you to meet because of the type of data or domain you are working in.
+
+### Sharing large volumes of models on the Hub
+
+Similarly to datasets, if you host models bigger than 300GB or if you plan on uploading a large number of smaller sized models (for instance, hundreds of automated quants) totalling more than 1TB, you will need to ask us to grant more storage. 
+
+To do that, to ensure we can effectively support the open-source ecosystem, please send an email with details of your project to models@huggingface.co.
+
+### Grants for private repositories
+
+If you need more model/ dataset storage than your allocated private storage for academic/ research purposes, please reach out to us at datasets@huggingface.co or models@huggingface.co along with a proposal of how you will use the storage grant.
```

Comment:
I wonder if we should make a grants@hf.co for this? - probably easier to manage?

Reply:
i think it's fine to keep two different emails (to route to ≠ people potentially)